src/content/docs/style-guide/writing-docs/writer-workflow/ui-install-guidelines.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: UI installation guidelines
+tags:
+  - Basic style guide
+  - Style guide quick reference
+---
+
+
+import styleguideDataSourceHeadingExample from 'images/style-guide_screenshot-full_data-source-heading-example.webp'
+
+import styleguideRepeatStageLabelinBody from 'images/style-guide_screenshot-full_repeat-stage-label-in-body.webp'
+
+import styleguideBlueCircledSteps from 'images/style-guide_screenshot-full_blue-circled-steps.webp'
+
+import styleguideDemonstrateLabelName from 'images/style-guide_screenshot-crop_demonstrate-label-name.webp'
+
+
+If you're reviewing or composing UI installation routines based on YAML files, follow these tips to keep your documents consistent and to streamline your work. You'll find both general principles and specific recomendations.
+
+## General principles [#principles]
+
+Writing these installation routines requires even more concise writing than general documentation on the main documentation site. Use these guidelines to help you keep your writing as streamlined as possible:
+
+* **First guiding principle:** We want users to be able to complete the installation process within the UI, without the docs or other external resources. In some cases, you'll encounter exceptions to this and have to link to third-party documentation because it is required for completion.
+* **Second guiding principle:** Less is more. Deletion should be your first impulse.
+
+
+
+## Heading information from data source file [#heading]
+
+At the very top of the installation stages and content pages is a description that comes from either the internal [data source files](https://source.datanerd.us/nr1-dev-experience/catalog-service-elixir/tree/master/apps/db/priv/data_sources) or the [community files](https://github.com/newrelic/newrelic-quickstarts/tree/main/data-sources).
```

Comment:
I think we want to avoid linking directly to source.datanerd.us.

Reply:
The community files repo is public, I believe, so fine to link to

Reply:
@paperclypse good catch! I'll drop this internal URL.